src/main/java/com/dougnoel/sentinel/elements/PageElement.java

```diff
@@ -267,6 +272,65 @@ public PageElement click() {
 		}
 		return this;
 	}
+	
+	/**
+	 * Drags the current element on top of the target element.
+	 * @param target PageElement the element the target is being dragged and dropped onto 
+	 * @return PageElement (for chaining)
+	 */
+	public PageElement dragAndDrop(PageElement target) {
+		new Actions(driver).dragAndDrop(this.toWebElement(), target.toWebElement()).build().perform();
+		return this;
+	}
+	
+	/**
+	 * Drags the current element on top of the target element via executing the drag and drop helper.js file
+	 * @param target PageElement the element the target is being dragged and dropped onto 
+	 * @return PageElement (for chaining)
+	 */
+	
+	public PageElement dragAndDropToTarget(PageElement target) {
+		dragAndDropToTarget(this.toWebElement(), target.toWebElement());
+		return this;
+	}
+	
+	/**
+	 * Reads the file from a given path.
+	 * @param path
+	 * @return 
+	 */
+	
+	public static String readFile(String path, Charset encoding)
```

Comment:
This method should be in the filemanagers namespace in a file named FileManager.java